src/clarity-angular/wizard/providers/page-collection.mock.ts

```diff
@@ -23,7 +23,8 @@ export class PageCollectionMock {
         return this._stepItemIdWasCalled;
     }
 
-    public previousPageIsCompleted(): boolean {
-        return true;
+    public _previousPageIsCompleted = true;
+    public previousPageIsCompleted(page: any = null): boolean {
```

Comment:
PageCollection needs to know which page you are asking about. You can ask it if any page's previous page is completed. So, not really a getter.